### PR TITLE
I have uploaded the collaborative projects.

### PR DESCRIPTION
**Pull request by norbiboy1**
This was my main project lists before Microverse.

### CODE REVIEW

**Comment by norbiboy1** [comment]
Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

# Good points
### Nice work
### Your project design looks amazing

# Aspects to improve
### Need to change all of the 2 projects stylesheet file
### Organize your code better
### You didn't configured the linters
### Your repository not documented in a professional way

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Just One More Thing 🔸

## Hi @FaranoshAmini 👋

Great work on making the changes requested ♻️ by a previous reviewer 👏.

You've done well implementing some of the requested changes 💯.

Though, There are some **few** issues 🐛 that you still need to work on to go to the next project but you are almost there! 🏃

## Don't Feel frustrated 🤔

I am sure you worked hard 🛠️ to make these changes and actually you have to be proud of them 🥇.

And I understand how much you wanted to get this project approved ✔️ (I was at your place one day 😉). But, don't worry, You still can do it! 💪.

Just apply the changes ♻️ down below 👇 and this time it is going to be approved ✅. Just make sure that you **don't miss any of the changes** requested or introduce any **new bugs** 🐛.

> **Note:** A good tip is to check the box next to every change after you implement it ☑️. And you can also check the full website to see if it is still working after you have implemented the changes.

*Please, reach to [me](#thank-you-for-your-time-🙏-and-call-me-if-you-need-help-or-clarification-☎️) for any help 🪝 or any question ❓. I am always happy to help you 😃.*

## Required Changes ♻️

- [ ] [REQUIRED] Please, make sure to use inline comments.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: microverse-repo1/index.html [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+code {
+	background-color: #CCC;
+	display: block;
+	padding: 20px;
+}
+@media screen and (min-width: 480px) {
+  body {
+    background-color: lightgreen;
+  }
+}
```
Comment (norbiboy1) [inline microverse-repo1/index.html:45]: Required changes #1: please use in a separate file the stylesheet, for best coding design practices.

File: microverse-repo1/index.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    </style>
+</head>
+
+<body>
+    <div class="main-body">
+    <nav id="navbar">
```
Comment (norbiboy1) [inline microverse-repo1/index.html:51]: Required changes #2: You should use only the header section for the main menu

File: microverse-repo2/index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+@media (max-width:500px) {
+    nav{
+      height:100%;
+      overflow:hidden;
+      display: flex;
+      align-items: center;
+      text-align: center;
+      flex-direction: column;
+    }
+  }
```
Comment (norbiboy1) [inline microverse-repo2/index.html:18]: Required changes #3: For the best experience, you need to use the stylesheet in a separate file. 👍

**Review by norbiboy1 — COMMENTED** [review]
Well done, Mr. Norbert. 💯.
You have to change some features, but you are already done it.
Had to congratulate you for your effort. 👍 😀

**Review by jaflih — APPROVED** [review]
Hi @norbiboy1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**